### PR TITLE
[JS] Topic : Data Types in JavaScript

### LINKED ISSUE #73: JS: Data Types in JavaScript
# Data Types in JavaScript
Discuss the built-in data types in JavaScript.

## Directory
```
Topics/5.Syntax/1.Data_Types
```

## Description
- Talk about the various data types in JavaScript.
- String, boolean, number, null, undefined, object.
- What is the `BigInt` type?
- How do we know the type of a particular value held in a variable?
  Hint: Using `typeof`.

## Hints
- Here's a lovely article to get you started: [Data Types in JavaScript](https://javascript.info/types).
- Check out the [W3C tutorial](https://www.w3schools.com/js/).

### PR DESCRIPTION
**Pull request by AbhinandanAdhikari**
<hr>

## Description 📜

Created a tutorial for JavaScript Data types.
Fixes #73 

<hr>

## Type of change 📝

- [ ] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev (Flutter, Java, Kotlin)
- [ ] Backend Dev (Java, .NET, Python)
- [ ] C/CPP
- [ ] Competitive Programming
- [ ] Cyber Security
- [ ] DSA
- [ ] Database
- [ ] Datascience with Python
- [ ] Datascience with R
- [ ] Frontend Dev (HTML, CSS, JS)
- [ ] Frontend Dev (React, Angular, Vue)
- [ ] Golang
- [ ] Interview Prep
- [ ] Java
- [x] JavaScript
- [ ] MERN
- [ ] Machine Learning
- [ ] Open Source
- [ ] Python
- [ ] Rust
- [ ] Statistics
- [ ] UI/UX

<hr>
 
## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by AbhinandanAdhikari** [comment]
@paramsiddharth  Sir please review this.

**Review by dakshsethi — CHANGES_REQUESTED** [review]

File: Web_Development/Javascript/Topics/5.Syntax/1.Data_Types/README.md [hunk 1]
```diff
@@ -1 +1,118 @@
...
+let age = 18;
+age = undefined; // now the value is undefined and type is also undefined
+```
+## 5. Null
+It represents null i.e no value at all.
+
+__Example:__
+```javascript
+let rollNo = null; // value here is unknown
+```
```
[the remark this answers is not in the document]
Comment (AbhinandanAdhikari) [reply Web_Development/Javascript/Topics/5.Syntax/1.Data_Types/README.md:66]: @dakshsethi  @paramsiddharth Sir, Made the changes, Please review it.

**Review by paramsiddharth — APPROVED** [review]

**Comment by paramsiddharth** [comment]
@Creatoon @dakshsethi Please review.

**Comment by AbhinandanAdhikari** [comment]
@Creatoon @dakshsethi  Sir, please review

**Comment by paramsiddharth** [comment]
@AbhinandanAdhikari Another PR with a document-based contribution to the same topic has been merged. I'm closing this PR.

Your documentation contains additional content with respect to the contribution by Aniket, so I encourage you to pull the changes, add content to the README file for Data Types, add your name to the contributors' list at the end of the file, and make a fresh PR. Best of luck!

**Pull request closed without merging**